src/libraries/System.Private.CoreLib/src/ILLink/ILLink.Substitutions.Shared.xml

```diff
@@ -1,5 +1,9 @@
 <linker>
   <assembly fullname="System.Private.CoreLib">
+    <type fullname="System.Diagnostics.Tracing.EventSource" feature="System.Diagnostics.Tracing.EventSource.IsSupported" featurevalue="false">
```

Comment:
NIT: did we do any sort of review for these feature switches? I ask because I wonder if it made more sense if we tried to have it so that all of them will default to substitute when the value is `true`. I'm only seeing these two here, but they get turned on by opposing booleans. Of course this is not super important, I'm just wondering if it was considered or if it mattered since most of the time folks that care about size will want to turn all of them on.

Reply:
@jkotas suggested using `Debugger.IsSupported` [here](https://github.com/dotnet/runtime/pull/37288#discussion_r433550303) and was fine with making the default value `true`. So I used the same logic here.

`GlobalizationMode.Invariant` has shipped a few versions already, so we can't change it.